packages/venia-concept/src/components/Checkout/__tests__/braintreeDropin.spec.js

```diff
@@ -0,0 +1,45 @@
+// NPM.
```

Comment:
Appreciate the thought, but let's not litter our files with these comments.

Eventually we'll write a lint rule if we want to make our conventions for import ordering more explicit and consistent.